### PR TITLE
feat(documentation): documentation contribution add more details

### PR DESCRIPTION
**Pull request by LilyYin17**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ Our full-stack web development and machine learning curriculum is completely fre
 ## Table of Contents
 
 - [Certifications](#certifications)
-- [The Learning Platform](#the-learning-platform)
+- [The Learning Platform - for users](#the-learning-platform)
```
Comment (samarmohan) [inline README.md:18]: It is definitely possible that developers would want to learn and normal users would want to contribute. Normal users could contribute to documentation and devs are always leveling up their skills.

File: README.md [hunk 2]
```diff
@@ -172,9 +172,14 @@ We appreciate any responsible disclosure of vulnerabilities that might impact th
 
 ### Contributing
 
-The freeCodeCamp.org community is possible thanks to thousands of kind volunteers like you. We welcome all contributions to the community and are excited to welcome you aboard.
+The freeCodeCamp.org community is possible thanks to thousands of kind volunteers like you. We welcome all contributions to the community and are excited to welcome you aboard. Here is an overview of what you can contribute:
```
Comment (samarmohan) [inline README.md:175]: `Here is an overview of what you can contribute:` should be `Here is an overview of what you can do to help:` This is clearer, cleaner, and flows better!

File: README.md [hunk 3]
```diff
@@ -172,9 +172,14 @@ We appreciate any responsible disclosure of vulnerabilities that might impact th
...
+- Answering coding questions on our community forum.
```
Comment (samarmohan) [inline README.md:176]: Fix tenses: `Answering` -> `Answer`

File: README.md [hunk 4]
```diff
@@ -172,9 +172,14 @@ We appreciate any responsible disclosure of vulnerabilities that might impact th
...
+- Give feedback on coding projects.
+- Translate freeCodeCamp.org's resources.
+- Contribute to our open source codebase on GitHub.
+- See the [issues](https://github.com/freeCodeCamp/freeCodeCamp/issues) for a list of proposed features (and known issues).
 
-> #### [Please follow these steps to contribute](https://contribute.freecodecamp.org).
+> #### [For more details, please follow these steps to contribute](https://contribute.freecodecamp.org).
```
Comment (samarmohan) [inline README.md:182]: The part before the comma doesn't flow or relate to the part after. I think you should keep it the same.

**Review by samarmohan — CHANGES_REQUESTED** [review]

**Comment by samarmohan** [comment]
Sorry if this sounded harsh, thank you so much though!

File: README.md (hunk 1 above)
Comment (Sembauke) [reply README.md:18]: This seems like an unnecessary change, and I agree with Samarmohan.

File: README.md (hunk 2 above)
Comment (Sembauke) [reply README.md:175]: As stated above this should be changed.

File: README.md (hunk 3 above)
Comment (Sembauke) [reply README.md:176]: Same here.

File: README.md (hunk 4 above)
Comment (Sembauke) [reply README.md:182]: I think this should not be changed, we want the contributors to read the contributing guide before contributing. This change seems to make it more optional.

**Comment by Sembauke** [comment]
Hey @LilyYin17,

Thanks for making these changes, they look great!

But can you apply the above changes suggested?

Have a nice day and happy coding.

File: README.md (hunk 2 above)
Comment (moT01) [reply README.md:175]: What about `Here are some ways you can contribute:`

**Comment by LilyYin17** [comment]
Did the revisions based on your suggestions, and pull the latest code.

**Comment by ShaunSHamilton** [comment]
@LilyYin17 Thank you, for making those changes. However, you appear to have pushed changes from other commits, as opposed to rebasing this branch with `main`. Could you please `revert` the last few commits to this branch?

**Comment by nhcarrigan** [comment]
I'm not sure these changes are needed. The README currently links to the contributing documentation, where the landing page has the list this PR adds with the relevant links to the documentation.

Rather than linking to the GitHub issues tab in the readme, I think the current flow of encouraging contributors to start with the docs is ideal.

**Comment by ShaunSHamilton** [comment]
@nhcarrigan Makes a decent point. After the recent suggestions, this PR adds information already on the main - what I call - _contributors page_ - where we actually want contributors to go.

As it stands, I find the amount of _stuff_ in the README too much.

Thank you, for all your time on this, and sorry about the to-and-fro, but I think we are going to close this PR.

**Pull request closed without merging**